packages/common/src/services/audius-backend/solana.ts

```diff
@@ -106,6 +106,39 @@ function isCreateUserBankIfNeededError(
   return 'error' in res
 }
 
+/**
+ * Returns the userbank account info for the given address and mint. If the
+ * userbank does not exist, returns null.
+ */
+export const getUserbankAccountInfo = async (
+  audiusBackendInstance: AudiusBackend,
+  { ethAddress: sourceEthAddress, mint = DEFAULT_MINT }: UserBankConfig = {}
+): Promise<AccountInfo | null> => {
+  const audiusLibs: AudiusLibs = await audiusBackendInstance.getAudiusLibs()
+  const ethAddress =
+    sourceEthAddress ?? audiusLibs.Account!.getCurrentUser()?.wallet
+
+  if (!ethAddress) {
+    throw new Error(
+      `getUserbankAccountInfo: unexpected error getting eth address`
+    )
+  }
+
+  if (!audiusLibs.solanaWeb3Manager!.doesUserbankExist({ ethAddress, mint })) {
```

Comment:
Doesn't this call the same fns as we're doing below? Can we just check it ourselves before returning to save on the solana RPC calls?